Dockerfile

```diff
@@ -1,12 +1,6 @@
-FROM gcc:11 as llama_builder 
+FROM ghcr.io/ggerganov/llama.cpp:light-19726169b379bebc96189673a19b89ab1d307659 as llama_builder 
 
-WORKDIR /tmp
-
-RUN git clone https://github.com/ggerganov/llama.cpp.git --branch master-d5850c5
-
-RUN cd llama.cpp && \
-    make && \
-    mv main llama
+RUN mv main llama
```

Comment:
You can probably remove this line. In the 2nd stage just copy the `main` binary into `/usr/bin/llama`

Reply:
@mcsgroi if you could please do this change while we're here in the PR, i'll approve & merge afterwards

Reply:
Addressed.